js/main.js

```diff
@@ -7,15 +7,12 @@ let didGameStart = false;
 
 
 
-
-let style = document.createElement('style');
-document.body.appendChild(style);
 let board = document.querySelector('.board');
-for (let i = 0; i < 7; i++) {
-    for (let j = 0; j < 7; j++) {
-        let d = document.createElement('div');
-        board.insertBefore(d, board.children[j + i * 7]);
-    }
+let boardModal = document.querySelector('.boardModal');
+for (let i = 0; i < 50; i++) {
+	let d = document.createElement('div');
+	console.log(board.innerHTML);
```

Comment:
Maybe drop this line from production code ✂️

Reply:
Will fix in some later PR - let's approve this one after TheoStefan's one so they don't have to do any merge conflict resolution